book/chapter-02.md

```diff
@@ -1,16 +1,29 @@
+
+Consequently, to ensure the safety of the Republic, the Founding Fathers sought to decentralize power and allow each and every individual in the country to own the most powerful weapon available at the time – a really crappy gun. This is what the Founding Fathers loved about it. They were a perfect decentralization of power. Moving forward a couple hundred years, we see many blockchains falling under this same spell. Satoshi wanted to design a system where the power to control money and information was as widely distributed as possible – a very, very big deal. 
+
+And just as the Founding Fathers turned to the gun, Satoshi turned to the CPU. Just like the guns of the 1790’s, CPUs are enormously powerful, yet widely distributed among the public, and few individuals have more than one or two. This means that this awesome power (the CPU), as Satoshi imagined, was reasonably distributed among the masses and controlled by no central entity. And for the first couple of years, Bitcoin was a virtual paradise of decentralization. Indeed, each year it became more and more decentralized. 
+But it is dangerous to decentralize based on technology, because technology changes. And so today, we might still call an AR-16 a gun, but from the perspective of the Founding Fathers, that would be like calling an armor plated cheetah a house cat. There wasn’t a single person alive in the late 1700’s, who, when they embraced the striking beauty of the 2nd amendment, considered something as deadly as modern weaponry. Returning to today, we see that blockchains like Bitcoin and others who rely on client-side validation through the use of miners have their own automatic rifle problem. It is called an ASIC. And an ASIC is unlike a CPU in that it is entirely useless to most folks (just as an AR-16 is today). As such, because few folks own them, ASICS are the opposite of decentralization. 
+
+Indeed, just as a few folks bought CPUs to mine, every purchase of an ASIC has been with the intention of rigging the system for the individual. The Founding Fathers and Satoshi both imagined a world where tech would create decentralization. But because they used tech, the opposite has happened. While we cross our fingers that one day, everyone will have an equal power to mine cryptocurrency. It does not seem likely. 
+
+The Founding Fathers were philosophers, not gun toters. And like Satoshi, they were chasing a dream of decentralization. But they both seemed to make the same mistake – they relied on technology. Decentralization, however, will always rely on people. 
```

Comment:
Its ironic that you say not gun toters. Timothy C. May one of the founding members of the cypherpunks loved guns and grew up with them - hardcore libertian

Reply:
Funny you say that too, found this: 
![image](https://user-images.githubusercontent.com/40184484/41289347-6e52aae8-6e17-11e8-9891-630e65fa8098.png)

Reply:
![image](https://user-images.githubusercontent.com/40184484/41289392-978bb2e2-6e17-11e8-9d9d-857914dfeb5b.png)

Reply:
written by Timothy May in 1994. That man was ahead of his time. 
Will think about that analogy I wrote about - might take it out. More effective ways to get the point across

Reply:
yeah, I thought the analogy was fine ~ just wanted to comment how May was a huge gunfreak lol